app/models/automate_workspace.rb

```diff
@@ -0,0 +1,17 @@
+class AutomateWorkspace < ApplicationRecord
+  include UuidMixin
+  belongs_to :user
+  belongs_to :tenant
+  validates :tenant, :presence => true
+  validates :user, :presence => true
+
+  def output=(hash)
+    if hash['workspace'].blank? && hash['state_var'].blank?
+      raise ArgumentError, "No workspace or state_var specified for edit"
+    end
+
+    self[:output] = (output || {}).deep_merge(hash)
```

Comment:
I believe it's preferable to use write_attribute over []= .  @jrafanie @chrisarcand Do you know what's the preferable way to do this?

Reply:
Oh duh...the preferable way is to use `super` 😝

Reply:
Prefer `output=` on the parent via `self.output = (output....`, if that's what you meant by `super`, yeah 😄